app/views/catalog/_document_media_object.atom.builder

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 xml.entry do
-  xml.title index_presenter(document).label(document_show_link_field(document))
+  xml.title document_presenter(document).heading
```

Comment:
I noticed that upstream they DRY this out by storing the presenter in a local variable:
`presenter = document_presenter(document)`